Makefile

```diff
@@ -396,3 +396,34 @@ watcher_cleanup: export CATALOG_IMG=${CATALOG_IMAGE}
 watcher_cleanup: ## Cleaning watcher operator via olm
 	bash ci/olm.sh
 	oc delete -f ci/olm.yaml
+
+KUTTL_SUITE ?= default
+KUTTL_NAMESPACE ?= watcher-kuttl-$(KUTTL_SUITE)
+KUTTL_SUITE_DIR ?= tests/kuttl/test-suites/$(KUTTL_SUITE)
+
+.PHONY: kuttl-test-prep
+kuttl-test-prep:
+	oc apply -k $(KUTTL_SUITE_DIR)/deps/ --timeout=120s
+	oc wait -n $(KUTTL_NAMESPACE) openstackcontrolplane openstack --for condition=Ready --timeout=300s
+
+.PHONY: kuttl-test-run
+kuttl-test-run:
+	oc kuttl test --v 1 --start-kind=false --config $(KUTTL_SUITE_DIR)/config.yaml
+
+.PHONY: kuttl-test
+kuttl-test: kuttl-test-prep kuttl-test-run
+
+.PHONY: kuttl-test-cleanup
+kuttl-test-cleanup:
+	# only cleanup if the $(KUTTL_NAMESPACE) exists
+	$(eval namespace_exists=$(shell oc get namespace $(KUTTL_NAMESPACE) --ignore-not-found -o name))
+	# We need to order the deletion. Simply deleting the namespace will
+	# result in errors in mariadb- and keystone-operator and then
+	# finalizer removal get stuck blocking the namespace deletion.
+	if [ "${namespace_exists}" != "" ]; then \
+		oc delete --wait=true --all=true -n $(KUTTL_NAMESPACE) --timeout=120s Nova; \
```

Comment:
nova -> watcher

this is the name of the top level CR we are creating.

because fo how we remove the finalised on resocues provided by keystone/infra operator we want to delete our CR first

then once we have cleaned up our reousces the rest can be done in one go.

Reply:
oops, bad copy/paste